docs/services/navigation/_index.md

```diff
@@ -554,3 +554,124 @@ obstacles = await my_nav.get_obstacles()
+
+### Compass Heading
+
+Compass heading measurements are read by the following {{< glossary_tooltip term_id="model" text="models" >}} of [movement sensor](/components/movement-sensor/):
+
+- [gps-nmea](/components/movement-sensor/gps/gps-nmea/)
+- [gps-nmea-rtk-pmtk](/components/movement-sensor/gps/gps-nmea-rtk-pmtk/)
+- [gps-nmea-rtk-serial](/components/movement-sensor/gps/gps-nmea-rtk-serial/)
+
+An example of a `Compass Heading` reading:
+
+``` go
+gps.GetCompassHeading{175}
+```
+
+Compass heading readings are taken through the movement sensor API's [`GetCompassHeading()`](/components/movement-sensor/#getcompassheading).
+If you want to read compass headings, [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
+Use compass heading readings to determine the *bearing* of your robot, or, the [cardinal direction](https://en.wikipedia.org/wiki/Cardinal_direction) that your robot is facing.
+
+### Orientation
+
+Orientation measurements are read by the following {{< glossary_tooltip term_id="model" text="models" >}} of [movement sensor](/components/movement-sensor/):
+
+- [imu-wit](/components/movement-sensor/imu/imu-wit/)
+- [imu-vectornav](/components/movement-sensor/imu/imu-vectornav/)
+
+An `Orientation` reading specifies the orientation of an object in 3D space as an "orientation vector", or, its position within the [cartesian coordinate system](https://en.wikipedia.org/wiki/Cartesian_coordinate_system) relative to some specific `origin` point that you, the user, need to choose and configure for your robot in Viam's [frame system](/services/frame-system/).
+
+An example of an `Orientation` reading:
+
+``` golang
+imuwit.GetOrientation{o_x: -0.58, o_y: -0.37, o:z 0.72, theta: 60.93}
+```
+
+Orientation readings are taken through the movement sensor API's [`GetOrientation()`](/components/movement-sensor/#getorientation).
+If you want to read orientation, [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
+Additionally, follow [these instructions](/services/frame-system/#configuration) to configure the geometries of each component of your robot within the [frame system](/services/frame-system/).
+
+### Angular Velocity
+
+The following {{< glossary_tooltip term_id="model" text="models" >}} of the [movement sensor](/components/movement-sensor/) component report angular velocity measurements:
+
+- [imu-wit](/components/movement-sensor/imu/imu-wit/)
+- [imu-vectornav](/components/movement-sensor/imu/imu-vectornav/)
+- [wheeled-odometry](/components/movement-sensor/wheeled-odometry/)
+- [gyro-mpu6050](/components/movement-sensor/mpu6050/)
+
+An example of a `AngularVelocity` reading:
+
+``` go
+imu.GetAngularVelocity{x: -47.9736, y: 142.639, z: -90.1489}
+```
+
+Angular velocity readings are taken through the movement sensor API's [`GetAngularVelocity()`](/components/movement-sensor/#getangularvelocity).
```

Comment:
```suggestion
Use the movement sensor API's [`GetAngularVelocity()`](/components/movement-sensor/#getangularvelocity) to get angular velocity readings.
```
or
```suggestion
To get angular velocity readings, use the movement sensor API's [`GetAngularVelocity()`](/components/movement-sensor/#getangularvelocity).
```
or 
```suggestion
To determine the speed and direction at which your robot is rotating, use the movement sensor API's [`GetAngularVelocity()`](/components/movement-sensor/#getangularvelocity).
```
If you like the third suggestion (my preference for succinctness and less redundancy), delete the "Use angular velocity.... sentence at the end of this paragraph.

Whichever way, these could all become more active.

Reply:
Tried to address this all like `The movement sensor API's [`GetPosition()`](/components/movement-sensor/#getposition) takes position readings.` lmk what you think! Agree, absoltuely needed to become more active voice-- I'm not the best at remembering the difference w/ active passive voice so if you still think the section could be better lmk with re-review and thanks!